src/sentry/static/sentry/app/views/settings/components/forms/type.tsx

```diff
@@ -26,12 +26,12 @@ export type FieldValue = any;
 
 type ConfirmKeyType = 'true' | 'false';
 
-// TODO(ts): A lot of these attribuets are missing correct types. We'll likely
+// TODO(ts): A lot of these attributes are missing correct types. We'll likely
 // need to introduce some generics in here to get rid of some of these anys.
 
 type BaseField = {
   label?: React.ReactNode | (() => React.ReactNode);
-  name?: string;
+  name: string;
```

Comment:
Making `name` required definitely makes things a lot easier and probably should have been done to begin with